src/assets/assets.controller.ts

```diff
@@ -176,4 +177,34 @@ export class AssetsController {
 
     res.status(HttpStatus.OK).json({ missing });
   }
+
+  @ApiOperation({ summary: 'Lists all verified asset metadata' })
+  @Get('verified_metadata')
+  @Header('Cache-Control', 's-maxage=60, stale-if-error=60')
+  async verifiedMetadata(
+    @Req() req: Request,
+    @Res() res: Response,
+  ): Promise<void> {
+    const lastUpdate = await this.assetsService.lastUpdate();
```

Comment:
This last update currently is looking at the asset table but I think we want to look at the assetMetadata table for the new one. I think the easiest thing to do would be to change the old endpoint to return the new data before this PR (new data database must be in mainnet and testnet databases before we do that). Once the old endpoint returns the new data we can re-use this function in the new endpoint